src/platform_impl/ios/uikit/press.rs

```diff
@@ -0,0 +1,95 @@
+use icrate::Foundation::{CGFloat, CGPoint, NSInteger, NSObject, NSTimeInterval};
+use objc2::encode::{Encode, Encoding};
+use objc2::{extern_class, extern_methods, mutability, ClassType};
+
+use super::{UIKey, UIResponder, UIWindow};
+
+#[cfg(feature = "i_dont_know_how_to_make_this_work")]
+use super::UIGestureRecognizer;
+use super::UIView;
+#[cfg(feature = "i_dont_know_how_to_make_this_work")]
+use objc2::rc::Id;
+
+extern_class!(
+    #[derive(Debug, PartialEq, Eq, Hash)]
+    pub(crate) struct UIPress;
+
+    unsafe impl ClassType for UIPress {
+        type Super = NSObject;
+        type Mutability = mutability::InteriorMutable;
+    }
+);
+
+extern_methods!(
+    unsafe impl UIPress {
+        #[method(locationInView:)]
+        pub fn locationInView(&self, view: Option<&UIView>) -> CGPoint;
+
+        #[method(key)]
+        pub fn key(&self) -> Option<&UIKey>;
+
+        #[method(type)]
+        pub fn type_(&self) -> UIPressType;
+
+        #[cfg(feature = "i_dont_know_how_to_make_this_work")]
+        #[method_id(gestureRecognizers)]
+        pub fn gesture_recognizers(&self) -> Id<NSArray<&UIGestureRecognizer>>;
```

Comment:
Like this ;)
```suggestion
        pub fn gesture_recognizers(&self) -> Id<NSArray<UIGestureRecognizer>>;
```

Reply:
Also, the docs says this is an optional:
```suggestion
        pub fn gesture_recognizers(&self) -> Option<Id<NSArray<UIGestureRecognizer>>>;
```